Makefile

```diff
@@ -32,6 +32,10 @@ ifneq ($(OS),Linux)
 endif
 ifneq ($(strip $(DOCKER_TAG)),)
 	docker build -t ${DOCKER_NAME} .
-	docker login -u ${DOCKER_USERNAME} -p "${DOCKER_PASSWORD}"
+ifneq (${DOCKER_USERNAME},)
+ifneq (${DOCKER_PASSWORD},)
+		docker login -u ${DOCKER_USERNAME} -p "${DOCKER_PASSWORD}"
```

Comment:
Indentation?

Reply:
Artifact of trying to do nested and indented `ifneq`s, will fix, thanks!